pkg/proxy/proxy.go

```diff
@@ -76,7 +76,7 @@ func (p *Proxy) StartProxy() *http.Server {
 
 	// listen concurrently to allow for graceful shutdown
 	log.Info(nil, "Starting the Proxy server...")
-	srv := &http.Server{Addr: ":" + ProxyPort, Handler: handler}
+	srv := &http.Server{Addr: ":" + ProxyPort, Handler: handler, ReadHeaderTimeout: 2 * time.Second}
```

Comment:
did you have any test failures that got resolved with this setting?

Reply:
yes, make lint was giving the following error, which got resolved:
``` pkg/proxy/proxy.go:79: File is not `gofmt`-ed with `-s` (gofmt)
        srv := &http.Server{Addr: ":" + ProxyPort, Handler: handler, ReadHeaderTimeout: 2*time.Second}```